elyra/pipeline/pipeline_definition.py

```diff
@@ -226,6 +227,15 @@ def subflow_pipeline_id(self) -> Pipeline:
         else:
             return None
 
+    @property
+    def get_component_links(self) -> List:
```

Comment:
should this function name include `input` since it only returns the inputs?

Reply:
Im fine with whatever, but since the links are always in the inputs structure, figured Id name it this